loader/CMakeLists.txt

```diff
@@ -184,11 +186,13 @@ if(WIN32)
     add_dependencies(loader-norm generate_helper_files loader_gen_files)
```

Comment:
Is there a particular reason why we're using object file libraries here? It looks like loader-norm could easily be a static library, which ends up being easier to handle: we can't "link" any libraries into an object file library, even for something like Vulkan::Headers, so this requires a different idiom to bring in the include paths.

Reply:
@lenny-lunarg I defer to you as Loader lead.